Libraries/Azure.psm1

```diff
@@ -2821,6 +2821,7 @@ function Add-AzureAccountFromSecretsFile {
 			#Verify if the user is Authorized to use the subscription.
 			$azContextResult = Set-AzContext -Subscription $XmlSecrets.secrets.SubscriptionID -ErrorAction SilentlyContinue
 			if ($azContextResult -and ($azContextResult.Subscription.Id -eq $XmlSecrets.secrets.SubscriptionID)) {
+				Set-Variable -Name AzAccountType -Value $azContextResult.Account.Type -Scope Global
```

Comment:
we should avoid adding new global variables.
Can we get the the AzContext by calling Get-AzContext directly in Invoke-IngestKustoFromTSQL function?

Reply:
Yes, we can, that's extra and additional  checking steps of doing ingestion (I think this will cost additional Rest API call to get all complex setting of context).  Ideally there should be somewhere to save global state of LISAv2, but there's no such central place.

Reply:
but anyway, I have updated following your comments. @lizzha